control-plane/tests/test_cli_inspection.py

```
@@ -208,6 +210,417 @@ def __init__(self, server_address: tuple[str, int], handler_class: type) -> None
+                    f"http://127.0.0.1:{servers[0].server_port}/intake/wazuh",
+                    data=oversized_payload,
+                    method="POST",
+                    headers={
+                        "Authorization": "Bearer reviewed-shared-secret",
+                        "Content-Type": "application/json",
+                        "X-Forwarded-Proto": "https",
+                    },
+                )
+
+                with self.assertRaises(error.HTTPError) as exc_info:
+                    request.urlopen(ingest_request, timeout=2)
```

Ruff (0.15.9)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)